contracts/Pool/PoolRegistry.sol

```diff
@@ -249,7 +249,7 @@ contract PoolRegistry is Ownable2StepUpgradeable {
             input.asset,
             comptroller,
             rate,
-            10**input.decimals,
+            10**(input.decimals - 8 + 18),
```

Comment:
I think `input.decimals` is the number of decimals of the VToken: 8, as you can see in the script number 10 [here](https://github.com/VenusProtocol/isolated-pools/pull/138/files#diff-402568a4acf31df6c7c204e6fd264f2985f0e229ffe0d2ecf5d7e6eded4de8dbR108), but what we need here is the number de decimals of the underlying token.

If we assume the underlying token is a ERC20 token we might get the number of decimal places in a dynamic way, invoking `decimals()` on the provided address. What do you think?

Reply:
You are right... What bothers me is why tests pass 🤔

Reply:
Already fixed